src/components/look/look-list/table/style-tag-list/index.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
+import { Button } from '@src/component/atoms';
+
+import { StyleTag } from '@src/types';
+import styleTagsData from '@src/data/style-tags';
+
+type StyleTagListProps = {
+  styleTags: StyleTag[];
+};
+
+const StyleTagList = ({ styleTags }: StyleTagListProps) => {
+  const handleSave = () => {
+    console.log('click Save');
+  };
+
+  const isChecked = (tag: StyleTag) => {
+    let flag = false;
+    styleTags.map((myTag) => {
+      if (myTag.id === tag.id) flag = true;
+    });
+    return flag;
+  };
+
+  return (
+    <>
+      <Wrapper>
+        {styleTagsData.map((tag: StyleTag) => (
+          <Button
+            key={tag.id}
+            title={tag.name}
+            onClick={() => {
+              //   tags.push(tag);
+              console.log(tag.id);
+            }}
+            style={{ margin: '0.4rem' }}
+            type={isChecked(tag) ? 'primary' : 'line'}
```

Comment:
```suggestion
            type={styleTags.find(styleTag => styleTag.id === tag.id) ? 'primary' : 'line'}
```

위와 같이 리팩토링 할 수 있습니다.

Reply:
0708ea9